src/ElectricPokemon.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private String specialAttack;
+    private String strongAgainst;
+
+    public ElectricPokemon(String name, int hp, int level, String food, String sound) {
+        super(name, hp, level, food ,sound, "electric");
+
+    }
+
+    public String getSpecialAttack() {
+        return specialAttack;
+    }
+
+    public String getStrongAgainst() {
+        return strongAgainst;
+    }
+
+    public List<String> getAttacks() {
+        return Arrays.asList("thunderPunch", "electroBall", "thunder", "voltTackle");
```

Comment:
Je kunt dit ook beter boven aan de klasse definieren als een constante, en die hier returnen. Dit geeft geen probleem, maar het is makkelijker als je alle constanten bij elkaar hebt staan, in plaats van hier in de getter iets en verderop nog eentje.